packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/EverstakeModal.tsx

```diff
@@ -42,15 +42,16 @@ export const EverstakeModal = ({ onCancel }: EverstakeModalProps) => {
                 margin={{ top: spacings.xs, bottom: spacings.lg }}
                 alignItems="stretch"
             >
-                <Banner icon="file" variant="info">
+                <Banner icon="fileFilled" variant="info">
                     <Translation
                         id="TR_STAKE_EVERSTAKE_MANAGES"
                         values={{
                             symbol: account?.symbol.toUpperCase(),
+                            t: text => <strong>{text}</strong>,
```

Comment:
Like with the Staking dashboard card, tooltip is not working for me :disappointed:

Reply:
The same reason as in the other case.